Example/Emission/AppDelegate.m

```diff
@@ -204,12 +216,12 @@ - (void)setupEmissionWithUserID:(NSString *)userID accessToken:(NSString *)acces
   emission.APIModule.artistFollowStatusAssigner = ^(NSString *artistID, BOOL following, RCTResponseSenderBlock block) {
     NSLog(@"Artist(%@).follow = %@", artistID, @(following));
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-      if (randomBOOL()) {
+//      if (randomBOOL()) {
         block(@[[NSNull null], @(following)]);
-      } else {
-        NSLog(@"Simulated follow request ‘failed’.");
-        block(@[RCTJSErrorFromNSError([NSError errorWithDomain:@"Artsy" code:42 userInfo:nil]), @(!following)]);
-      }
+//      } else {
+//        NSLog(@"Simulated follow request ‘failed’.");
+//        block(@[RCTJSErrorFromNSError([NSError errorWithDomain:@"Artsy" code:42 userInfo:nil]), @(!following)]);
+//      }
```

Comment:
These simulated network errors were getting real tedious. We should re-enable that at some point, but preferably in a way where the user of the example app knows that a network error will be simulated, otherwise we forget the these happen and are baffled when things seemingly suddenly break.